tests/test_other.py

```diff
@@ -8102,6 +8102,12 @@ def test_no_legalize_js_ffi(self):
       e_legalstub_i32 = re.search('\(func.*\$legalstub\$dyn.*\(type \$\d+\).*\(result i32\)', text)
       assert i_legalimport_i64, 'legal import not generated for invoke call'
       assert e_legalstub_i32, 'legal stub not generated for dyncall'
+    args = ['-s', 'LEGALIZE_JS_FFI=0', '-s', 'SIDE_MODULE=1', '-O3', '-s', 'DISABLE_EXCEPTION_CATCHING=0']
+    print(args)
+    cmd = [PYTHON, EMCC, path_from_root('tests', 'other', 'noffi-side.cpp'), '-g', '-o', 'side.wasm'] + args
+    print(' '.join(cmd))
```

Comment:
I do see this fail in the node command, with `function signature mismatch` - but this emcc command is never executed here. So `side.wasm` does not exist. Is that the cause of the problem?

Reply:
@kripken  sorry about that. i have pushed a fix to the test so it fails properly now. the error is 
`exception thrown: TypeError: invalid type,TypeError: invalid type`